enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackupExtensionFactory.java

```diff
@@ -85,15 +87,19 @@ public OnlineBackupExtensionFactory()
     @Override
     public Lifecycle newInstance( KernelContext context, Dependencies dependencies )
     {
-        return new OnlineBackupKernelExtension( dependencies.getConfig(), dependencies.getGraphDatabaseAPI(),
-                dependencies.logService().getInternalLogProvider(), dependencies.monitors(),
-                dependencies.neoStoreDataSource(),
-                dependencies.checkPointer(),
-                dependencies.transactionIdStoreSupplier(),
-                dependencies.logicalTransactionStoreSupplier(),
-                dependencies.logFileInformationSupplier(),
-                dependencies.fileSystemAbstraction(),
-                dependencies.pageCache(),
-                dependencies.storeCopyCheckPointMutex() );
+        if ( !isClusterInstance( context ) )
+        {
+            return new OnlineBackupKernelExtension( dependencies.getConfig(), dependencies.getGraphDatabaseAPI(),
+                    dependencies.logService().getInternalLogProvider(), dependencies.monitors(), dependencies.neoStoreDataSource(), dependencies.checkPointer(),
+                    dependencies.transactionIdStoreSupplier(), dependencies.logicalTransactionStoreSupplier(), dependencies.logFileInformationSupplier(),
+                    dependencies.fileSystemAbstraction(), dependencies.pageCache(), dependencies.storeCopyCheckPointMutex() );
+        }
+        return new LifecycleAdapter();
+    }
+
+    private static boolean isClusterInstance( KernelContext kernelContext )
```

Comment:
Possibly call this isCausalClusterInstance for least confusion

Reply:
Resolved